docs/network-setup.md

```diff
@@ -96,6 +96,85 @@ your environment. For testing, you can add a public DNS server to
 nameserver 8.8.8.8
 ```
 
+## [Advanced] Setting Up a Bridge Interface
+
+<!-- markdownlint-disable-next-line MD024 -->
+### On The Host
+
+1. Create a bridge interface
+
+   ```bash
+   sudo ip link add name br0 type bridge
+   ```
+
+1. Add tap interface [created above](#on-the-host) to the bridge
+
+   ```bash
+   sudo ip link set dev tap0 master br0
+   ```
+
+1. Define an IP address in your network for the bridge.
+
+   For example, if your gateway were on `192.168.1.1` and you wanted to use this
+   for getting dynamic IPs, you would want to give the bridge an unused IP address
+   in the `192.168.1.0/24` subnet.
+
+   ```bash
+   sudo ip address add 192.168.1.7/24 dev br0
+   ```
+
+1. Add firewall rules to allow traffic to be routed to the guest
+
+   ```bash
+   sudo iptables -t nat -A POSTROUTING -o br0 -j MASQUERADE
+   ```
+
+### On The Guest
+
+1. Define an unused IP address --- in the bridge's subnet e.g., `192.168.1.0/24`
+for the guest's ethernet interface. This is also [documented above](#on-the-host).
```

Comment:
Bring this line to under the one above and the CI should pass.

Reply:
@dianpopa sorry I don't understand. Are you suggesting to removing the space between the "On The Guest" section header and the first numbered bullet (i.e., "Define an unused IP...")?

I'm confused because I thought I saw in CI that it expected whitespace surrounding bullets.

Reply:
@dianpopa I gave it a shot! https://github.com/firecracker-microvm/firecracker/pull/2688/commits/a34e108c70fdd0bc784430b8ed619a083cc297e9

If you re-run CI, we can see and get it working :)

Reply:
@mccurdyc you can easily test the markdown linter on your local or remote setup, no need to rely on running our full CI upstream:
```bash
tools/devtool test -- integration_tests/style/test_markdown.py
```

Or you could run all "style" tests:
```bash
tools/devtool test -- integration_tests/style/
```

Reply:
Wow I feel silly now. I forgot that I hard wrapped that line because there was a line-length markdown error. Now your original comment makes sense and I've pushed up a commit that should address it. I've gone ahead and squashed my commits. Thank for being patient with me @acatangiu @dianpopa